src/events/eventToAction.js

```diff
@@ -140,6 +140,7 @@ const stream = (state: GlobalState, event: Object): EventStreamAction => ({
   type: opToActionStream[event.op],
 });
 
+// $FlowFixMe
```

Comment:
Would you add a few words saying what the error is here and why it's hard to fix? I'd like us to start having some kind of explanation for all our `$FlowFixMe`s.

Reply:
Added details.

Reply:
Thanks! This is helpful.